I tried to build Blinky as specified in the script but I got the following error:
```
user at hostname in /path/to/mbed-os-example-blinky on feature-cmake
$ mbedtools configure -t GCC_ARM -m K64F
mbed_config.cmake has been generated and written to '/path/to/mbed-os-example-blinky/.mbedbuild'
user at hostname in /path/to/mbed-os-example-blinky on feature-cmake
$ cd build/
user at hostname in /path/to/mbed-os-example-blinky/build on feature-cmake
$ cmake .. -GNinja -DMBED_PROFILE=develop
CMake Error at mbed-os/cmake/app.cmake:84 (add_custom_command):
  add_custom_command called with wrong number of arguments.
Call Stack (most recent call first):
  CMakeLists.txt:10 (include)
CMake Error at mbed-os/cmake/app.cmake:85 (add_custom_command):
  add_custom_command called with wrong number of arguments.
Call Stack (most recent call first):
  CMakeLists.txt:10 (include)
-- Configuring incomplete, errors occurred!
See also "/path/to/mbed-os-example-blinky/build/CMakeFiles/CMakeOutput.log".
See also "/path/to/mbed-os-example-blinky/build/CMakeFiles/CMakeError.log".
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -108,18 +108,25 @@ matrix:
       stage: "CMake"
       name: "CMake blinky example"
       env: NAME=cmake_test
+      language: python
+      python: 3.8
       install:
         # Install gcc
         - source_pkg gcc
         - arm-none-eabi-gcc --version
-        - pip install --user --upgrade cmake
+        - pip install --upgrade cmake
+        - pip install --upgrade mbed-tools
+        # We use manual clone, with depth and single branch = the fastest
+        # Because of this, we need to create .mbed file as the current tools require it
         - >-
          git clone --depth=1 --single-branch --branch feature-cmake https://github.com/ARMmbed/mbed-os-example-blinky.git;
          cd mbed-os-example-blinky;
          git clone --depth=1 --single-branch --branch feature-cmake https://github.com/ARMmbed/mbed-os.git;
+         touch .mbed;
+         mbedtools configure -t GCC_ARM -m K64F;
          mkdir -p build;
       script:
-        - cd build && cmake .. -GNinja && ninja
+        - cd build && cmake .. -GNinja -DMBED_PROFILE=develop && cmake --build .
 
     ### Docs Tests ###
     - &docs-vm

